pkg/imagefilter/formatter.go

```diff
@@ -0,0 +1,93 @@
+package imagefilter
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+)
+
+// OutputFormat contains the valid output formats for formatting results
+type OutputFormat string
+
+const (
+	OutputFormatDefault OutputFormat = ""
+	OutputFormatText    OutputFormat = "text"
```

Comment:
Fwiw, we could easily support `text-fmt` as formatter here to allow comamndline things like `--output=text --format={{.Distro.Name}}` but let's YAGNI this for now (unless we/our users need it)

Reply:
We could also have a "json-experimental" or something formatter that would give us a lot more details without API gurantees (again YAGNI, but we are fleixble enough here)

Reply:
I think it's good to experiment with a few things and see what works, what feels good.  We're not releasing anything (yet), but we should start using it in CI and our own day-to-day.